opal/corelib/string.rb

```diff
@@ -1841,6 +1841,10 @@ def unpack(format)
   def unpack1(format)
     raise "To use String#unpack1, you must first require 'corelib/string/unpack'."
   end
+
+  def frozen?
+    `typeof self === 'string'`
+  end
```

Comment:
Could be removed... the more I think about it, the more I think we should remove it but waiting Elia's input 😉

Reply:
On face value it makes sense to me, what considerations are nudging you toward removing it?

Reply:
I felt like we were twisting the definition of `frozen?` but you made a good point about `frozen_string_literal`. Let's keep it 🤝